cmake-proxies/cmake-modules/AudacityPackageTarball.cmake

```diff
@@ -0,0 +1,49 @@
+# Defines zip-package and targz-package targets
+# cmake --install won't invoke build, it is expected, that required configuration is already built
```

Comment:
Couldn't we just use the CPack `ZIP` and `TGZ` generators?

Reply:
You're not actually using `zip-package` and `targz-package` anywhere now, are they?

Reply:
I have removed both targets